README.md

```diff
@@ -112,10 +112,40 @@ Once Conjur is configured, you can create a new API client by providing a `login
 Conjur::API.new_from_key login, api_key
 ```
 
-Note that if you are connecting as a [Host](http://developer.conjur.net/reference/services/directory/host), the login should be 
+Note that if you are connecting as a [Host](http://developer.conjur.net/reference/services/directory/host), the login should be
 prefixed with `host/`. For example: `host/myhost.example.com`, not just `myhost.example.com`.
 
-# Development
+
+# Development (V5)
+To develop and run tests against Conjur V5, use the `start` and `stop` scripts in the `dev` folder. The start script brings up an open source Conjur (and Postgres database), CLI container, and a "work" container, with the gem code mounted into the working directory.
+
+To begin:
+```sh
+$ cd dev
+$ ./start
+...
+root@9df0ac10ada2:/src/conjur-api#
+```
+You'll be dropped into development container upon completion. From there, install the development gems:
+
+```sh
+root@9df0ac10ada2:/src/conjur-api# bundle
+```
+
+Tests can be run with:
+```sh
+root@9df0ac10ada2:/src/conjur-api# cucumber features
```

Comment:
Running `cucumber features` did not work for me. I needed on additional change in the `docker-compose.yaml` and I needed to run the command:
```sh
bundle exec rake jenkins_init jenkins_cucumber_v5
```
to run the cucumber tests in the dev container.